enterprise/frontend/src/embedding-sdk/components/public/StaticDashboard/StaticDashboard.tsx

```diff
@@ -0,0 +1,85 @@
+import { pick } from "underscore";
+
+import {
+  DEFAULT_EMBED_DISPLAY_OPTIONS,
+  useDashboardFullscreen,
+  useDashboardRefreshPeriod,
+  useEmbedTheme,
+  useRefreshDashboard,
+} from "metabase/dashboard/hooks";
+import type {
+  EmbedDisplayParams,
+  RefreshPeriod,
+} from "metabase/dashboard/types";
+import { isNotNull } from "metabase/lib/types";
+import { PublicDashboard } from "metabase/public/containers/PublicDashboard/PublicDashboard";
+import { Box } from "metabase/ui";
+import type { DashboardId, ParameterId } from "metabase-types/api";
+
+export const StaticDashboard = ({
+  dashboardId,
+  parameterValues,
+  refreshPeriod: initialRefreshPeriod = null,
+  bordered,
+  titled,
+  theme: userTheme,
+  font,
+  hideDownloadButton,
+  hideParameters,
+}: {
+  dashboardId: DashboardId;
+  parameterValues: Record<ParameterId, string | string[] | null | undefined>;
```

Comment:
could/should the type be the same as this one? 
https://github.com/metabase/metabase/blob/911e76ba524cc281c5d469ec70a035ac5b6becaa/frontend/src/metabase/public/components/EmbedFrame/EmbedFrame.tsx#L51

At glance, they're both very similar, the only difference I can see is `undefined` vs `null`. I don't know why they need to be different.

Reply:
Yep. I've changed it to `Query` just now